osi_common.proto

```diff
@@ -325,18 +325,17 @@ message BaseMoving
     //
```

Comment:
Please add to line 265 & 319: The relative orientation of the stationary object w.r.t. its parent frame, noted in the parent frame. The orientation becomes global/absolute if the parent frame is inertial .